README.md

```diff
@@ -50,15 +50,15 @@ Snuba exposes an HTTP API (default port: `1218`) with the following endpoints.
 
 Settings are found in `settings.py`
 
-- `CLICKHOUSE_SERVER` : The endpoint for the clickhouse service.
+- `CLICKHOUSE_HOST` : The hostname for the clickhouse service.
 - `REDIS_HOST` : The host redis is running on.
 - `DATASET_MODE` : If "local" runs Clickhouse local tables instead of distributed ones.
 
 ## Tests
 
     pip install -e .
 
-    export CLICKHOUSE_SERVER=127.0.0.1:9000
+    export CLICKHOUSE_HOST=127.0.0.1:9000
```

Comment:
Why not just removing this line and letting the test pick up the default setting ?

Reply:
Fine with me, also just realized this included the port. Oops!

Reply:
Fixed this with b9eb2b4.